case-lib/lib.sh

```diff
@@ -35,7 +35,7 @@ fi
 
 # setup SOFCARD id
 if [ ! "$SOFCARD" ]; then
-    SOFCARD=$(grep 'sof-[a-z]' /proc/asound/cards|awk '{print $1;}')
+    SOFCARD=$(grep -v 'sof-probes' /proc/asound/cards | grep 'sof-[a-z]' | awk '{print $1;}')
```

Comment:
@aiChaoSONG im my opinion this should be handled slightly differently. We should run sof-test for all sof-* cards. This is coming soon too after the auxiliary bus patch is accepted. And anyway with sof-probes, there will really be nothing to do (just yet) becaue it doesnt add any pcm devices, only compress devices.

Reply:
We cannot acquire the PCM card mapping from TPLG, right? @xiulipan We need to carefully deal with this change.

Reply:
@aiChaoSONG @ranj063 Yes, I would say this patch is only a simple workaroud for the PR2501.
We have some coming change that will spilt existing PCMs to different card. Let's have some sync up meeting first to decided what should be done for the update.